src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private Storage storage;
+    private Ui ui;
+    private String filePath;
+
+    public Duke(String filePath) {
```

Comment:
Perhaps need a descriptive header comments for the Duke method? 🤔
And I noticed the same issue in several other places below.